add_tokens.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+def load_data():
+    try:
+        data = pd.read_csv('additiona_tokens.csv',index_col=[0])
+    except:
+        print('no data found')
+        data = pd.DataFrame()
+    return data
+
+def get_additional_tokens(dex):
+    data = load_data()
+    to_add = pd.Series(dtype=float)
+    if dex in data.index:
+        for i in data.loc[dex]:
+            j = i +'<>weth'
+            to_add.loc[j] = 1e7
+    return to_add
+
+def add_tokens(dex, data):
+    to_add = get_additional_tokens(dex)
+    return data.append(to_add)
```

Comment:
Since this function is pure it's actually amenable to a small test. I know we don't have tests in this package yet but if you want to add one it might be easy; I'd even be willing to set up CI on it.

Reply:
Agreed and totally welcome that. Let's start writing tests for this repo.